<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>janela_login</class>
 <widget class="QDialog" name="janela_login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Wefix Store™</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imgs.qrc">
    <normaloff>:/Imgs/UI/PNG/icon.png</normaloff>:/Imgs/UI/PNG/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#janela_login{
background-color: rgb(52, 73, 94);
color:#fff;
font: 12pt &quot;Roboto&quot;;
}

.QMessageBox
{
	background-color: rgb(52, 73, 94);
	text-color: rgb(255, 255, 255);
	font: 12pt &quot;Roboto&quot;;
}

.QLabel
{	
	color: rgb(255, 255, 255);
}

.QPushButton
{
	
	background-color: rgb(52, 152, 219);
	border-radius:1px;
	color: rgb(255, 255, 255);
	font: 12pt &quot;Roboto&quot;;
	width: 50px;
	height: 30px;
}
.QPushButton:hover
{
	background-color: rgb(41, 128, 185);
}


.QLineEdit
{
	background-color:#7f8c8d;
	color:2c3e50;
}


.QLineEdit:focus
{
	background-color:#bdc3c7;
	text-color:2c3e50;
}

.QLineEdit:hover
{
	background-color:#bdc3c7;
	text-color:#2c3e50;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Roboto&quot;;</string>
   </property>
   <property name="text">
    <string>LOGIN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>340</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#btn_login
{
	background:#3498db;
	border-radius:1px;
}

#btn_login:hover
{
	background:#2980b9;
}</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/Imgs/UI/Icones/Store/cadeado_aberto.png</normaloff>:/Imgs/UI/Icones/Store/cadeado_aberto.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_fechar">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>340</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#btn_fechar
{
	background-color:#e74c3c;
	color:#fff;
	border-radius: 1px;
}
#btn_fechar:hover
{
	background-color:#c0392b;
}
</string>
   </property>
   <property name="text">
    <string>Fechar</string>
   </property>
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/Imgs/UI/Icones/Store/fechar.png</normaloff>:/Imgs/UI/Icones/Store/fechar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_login">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>240</y>
     <width>310</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">.QLineEdit:focus
{
	background-color:#bdc3c7;
	text-color:2c3e50;
}

.QLineEdit:hover
{
	background-color:#bdc3c7;
	text-color:#2c3e50;
}</string>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Usuário</string>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::VisualMoveStyle</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_senha">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>290</y>
     <width>310</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Senha</string>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::VisualMoveStyle</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_info">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(231, 76, 60);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Imgs/UI/Tela.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>300</width>
     <height>160</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
margin-left: auto;
margin-right: auto;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imgs.qrc">:/Imgs/UI/msg_inicial.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>25</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imgs.qrc">:/Imgs/UI/PNG/bem_vindo.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_info">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>570</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: none;
border-radius:1px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/Imgs/UI/Icones/Store/info.png</normaloff>:/Imgs/UI/Icones/Store/info.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>250</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imgs.qrc">:/Imgs/UI/Icones/Store/usuario.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>300</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imgs.qrc">:/Imgs/UI/Icones/Store/cadeado_fechado.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>le_login</tabstop>
  <tabstop>le_senha</tabstop>
  <tabstop>btn_login</tabstop>
  <tabstop>btn_fechar</tabstop>
 </tabstops>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
